examples/pytorch_lightning_simple.py

```diff
@@ -15,14 +15,14 @@
 import os
 import shutil
 
-from packaging import version
 import pytorch_lightning as pl
-from pytorch_lightning import Callback
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
-from torch.optim import Adam
 import torch.utils.data
+from packaging import version
+from pytorch_lightning import Callback
+from torch.optim import Adam
 from torchvision import datasets
 from torchvision import transforms
```

Comment:
https://github.com/PyCQA/isort#controlling-how-isort-sections-from-imports says:

> By default isort places straight (import y) imports above from imports (from x import y):

```python
import b
from a import a  # This will always appear below because it is a from import.
```

> However, if you prefer to keep strict alphabetical sorting you can set force sort within sections to true. Resulting in:

```python
from a import a  # This will now appear at top because a appears in the alphabet before b
import b
```

Reply:
I don't have a strong opnion about this, but the default import-comes-first sorting rule looks much cleaner and straightforward.

Reply:
Totally agree with you. It makes the sorting rule simple. I'd like to know other reviewers' opinions.

Reply:
I think it is kind for developers to add an explanation of the  "import-comes-first" rule in `CONTRIBUTING.md`.

Reply:
I'll update `CONTRIBUTING.md` in a follow-up.